eng/Packaging.props

```diff
@@ -27,7 +27,7 @@
     <PackageVersion Condition="'$(PackageVersion)' == ''">4.7.0</PackageVersion>
     <!-- major.minor.release version of the platforms package we're currently building
          Pre-release will be appended during build -->
-    <PlatformPackageVersion>3.1.0</PlatformPackageVersion>
+    <PlatformPackageVersion>3.1.1</PlatformPackageVersion>
```

Comment:
@safern Does this require rebuilding the package index with this PR? Or is that something that gets handled automagically?

Reply:
@safern Ping?

Reply:
@ericstj...@safern is out

Reply:
I believe this is sufficient.  @Anipik when do we typically add the stable version to the index?

Reply:
we add it after the package has been pushed to the nuget. i.e on the day of release with the branding change.